cmd/containerd-shim-runc-v2/task/service.go

```diff
@@ -24,6 +24,10 @@ import (
 	"os"
 	"sync"
 
+	"github.com/containerd/containerd/v2/api/types"
+	"google.golang.org/protobuf/types/known/emptypb"
+	"google.golang.org/protobuf/types/known/timestamppb"
+
```

Comment:
I think this empty line can be removed.

Reply:
I think meant to divide containerd imports from externals.
Updated.